conf/validation-config/EMI/ers-emi-taxable-validation.conf

```diff
@@ -101,7 +101,7 @@ ers-emi-taxable-validation-config {
     {
       column = "H"
       cellName = "8. PAYE reference"
-      mandatory = false
+      mandatory = true
       error: {
           id = "error.8"
           errorMsg = ${validation-types.paye.errorMsg}
```

Comment:
Just to confirm, here the error msg/rule need no updating?

Reply:
Yes this is correct. I double checked this with @DannyPennington97 and Muhammed and due to the fact this PAYE reference isn't specific to emp like the rest of the changes, can just be left the same for now